views/deposits.slim

```diff
@@ -2,9 +2,10 @@ h2 Deposits
 section id="deposit_form"
   label for="name" Name:
   br
```

Comment:
Rather than have a `br` tag you can wrap each element into a `div`

Reply:
Alright.